packages/dds-components/src/hooks/useRoveFocus.mdx

```diff
@@ -0,0 +1,16 @@
+import { Canvas, Controls, Meta } from '@storybook/blocks';
+import { ComponentLinkRow } from '@norges-domstoler/storybook-components';
+import * as RoveStories from './useRoveFocus.stories';
+
+<Meta of={RoveStories} />
+
+# useRoveFocus
+
+<ComponentLinkRow
+  docsHref="https://design.domstol.no/987b33f71/p/0202f1-userovefocus"
+  githubHref="https://github.com/domstolene/designsystem/blob/main/packages/dds-components/src/hooks/useRoveFocus.tsx"
+  withBottomSpacing
+/>
```

Comment:
Du får en død lenke for Storybook siden den er påkrevd i `<ComponentLinkRow>`:

![image](https://github.com/user-attachments/assets/b1299ba3-2d50-4ddf-884f-57fb549e2ae2)

Du kan gå inn på fila `packages/storybook-components/src/StorybookLinkRow.tsx` og gjøre prop `storybookHref` ikke påkrevd for `<ComponentLinkRow>`.